src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2519,6 +2519,7 @@ static WellKnownMembers()
                 0,                                                                                                          // Arity
                     (byte)SignatureTypeCode.TypeHandle, (byte)WellKnownType.System_Runtime_GCLatencyMode,                   // Field Signature
 
+
```

Comment:
I had those empty lines on purpose to provide visual structure. I'll remove them.